src/javascripts/Services/referencedByViewColumnsService.js

```diff
@@ -0,0 +1,114 @@
+ngapp.service('referencedByViewColumnsService', function(settingsService) {
+    let service = this,
+        settings = settingsService.settings;
+
+    let longNameColumn = {
```

Comment:
It's interesting that xEdit displays LongName in a column, signature in another column, and file in another.  I feel like we could maybe display different columns by default.

Perhaps something like: Signature, FormID, EditorID, Name, File.  Where Name would display similar to how it displays on the tree view (`FullName` or `PlacementName`).  That said, using such a similar column loadout could make it confusing with the tree view, so we should experiment.

Because we don't have native sorting OR node expansion on this view we should be able to make every column optional from the edit columns modal.